contributor_docs/webgl_contribution_guide.md

```diff
@@ -0,0 +1,161 @@
+# WebGL Contribution Guide
+
+If you're reading this page, you're probably interested in helping work on WebGL mode. Thank you, we're grateful for your help! This page exists to help explain how we structure WebGL contributions and to offer some tips for making changes.
+
+
+## Resources
+
+- Read our [p5.js WebGL architecture overview](webgl_mode_architecture.md) to understand how WebGL mode differs from 2D mode. This will be a valuable reference for some implementation specifics for shaders, strokes, and more.
```

Comment:
@limzykenneth This doc links to the other doc. Right now I have this linking to an md file with a relative path, but I'm not sure if that will get converted when put onto p5js.org. Should we just do this and add link conversion to the requirements when parsing md(x) files for the site?

Reply:
Yeah let's leave it as is for now if the link is currently working. We'll review if we need to change it when the website has more structure.